beps/bep_0052_torrent_creator.py

```diff
@@ -229,8 +232,15 @@ def create(self, tracker, hybrid=True):
             info[b'pieces'] = self.pieces
             try: info[b'files'] = self.files
             except AttributeError: info[b'length'] = self.length
+        self.info = info
         return {b'announce': tracker, b'info': info, b'piece layers': {f.root: f.piecesv2 for f in self.piece_layers if f.length > self.piece_length}}
-    
+
+    def info_hash_v2(self):
+        return binascii.hexlify(sha256(encode(self.info)).digest()).decode('ascii')
```

Comment:
there's no need to use binascii here, just call `hexdigest()` instead of `digest()`